examples/with-thirdparty/with-thirdparty/LoginScreen/LoginScreenViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  LoginScreenViewController.swift
+//  with-thirdparty
+//
+//  Created by Nemi Shah on 10/11/23.
+//
+
+import UIKit
+import GoogleSignIn
+import AppAuth
+
+class LoginScreenViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    @IBAction func onGoogleCliked() {
+        GIDSignIn.sharedInstance.signIn(withPresenting: self) { signInResult, error in
+            guard error == nil else { return }
+
+            guard let authCode: String = signInResult?.serverAuthCode as? String else {
+                print("Google login did not return an authorisation code")
+                return
+            }
+            
+            let url = URL(string: Constants.apiDomain + "/auth/signinup")
+            var request = URLRequest(url: url!)
+            request.httpMethod = "POST"
+            
+            let data = try! JSONSerialization.data(withJSONObject: [
+                "thirdPartyId": "google",
+                "redirectURIInfo": [
+                    // For native flows we do not have a redirect uri
+                    "redirectURIOnProviderDashboard": "",
+                    "redirectURIQueryParams": [
+                        "code": authCode
```

Comment:
the docs mention sign in with google gets the access and id token on the frontend itself. So stick to that please. make sure that your code snippets here are fully consistent with the docs.

Reply:
Theres to options here:
1. We update the ios example app to match the docs but be different from the Android flow
2. We update the ios docs to use the authorisation code and make it consistent with Android

Im fine with either